api/v1alpha1/staticroute_types.go

```diff
@@ -0,0 +1,74 @@
+/* Copyright © 2021 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type StaticRouteStatusCondition string
+
+const (
+	StaticRouteReady    StaticRouteStatusCondition = "Ready"
+	StaticRouteNotReady StaticRouteStatusCondition = "NotReady"
+)
+
+// StaticRouteCondition defines condition of StaticRoute.
+type StaticRouteCondition struct {
+	// Type defines condition type.
+	Type StaticRouteStatusCondition `json:"type"`
+	// Status defines status of condition type, True or False.
+	Status corev1.ConditionStatus `json:"status"`
+	// Reason shows a brief reason of condition.
+	Reason string `json:"reason,omitempty"`
+	// Message shows a human readable message about condition.
+	Message string `json:"message,omitempty"`
+}
+
+// StaticRouteSpec defines static routes configuration on Tier-0/Tier-1 or on VPC
+type StaticRouteSpec struct {
+	// Specify next hop routes for network
+	StaticRoutes []StaticRouteItem `json:"routes"`
+}
+
+type StaticRouteItem struct {
+	// Unique route name to indicate the specific node in specific tkc
+	Name string `json:"name"`
+	// Next hop gateway IP address
+	Target string `json:"target"`
+	// Specify network address in CIDR format
+	Destination string `json:"destination"`
```

Comment:
// +kubebuilder:validation:Format=cidr
We can add this crd validation.

Reply:
done